Packs/FeedAlienVaultReputation/TestPlaybooks/AlienVaultReputationFeed_Test.yml

```diff
@@ -0,0 +1,202 @@
+    quietmode: 0
+  "2":
+    id: "2"
+    taskid: bf52c495-214f-4b9d-87b6-a66e63f1a954
+    type: condition
+    task:
+      elasticcommonfields: {}
+      id: bf52c495-214f-4b9d-87b6-a66e63f1a954
+      version: -1
+      name: Assert Output
+      description: Asserts output.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: AlienVaultReputationFeed
+                accessor: Indicator.Value
+            iscontext: true
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: AlienVaultReputationFeed
+                accessor: Indicator.Type
+            iscontext: true
+          right:
+            value:
+              simple: IP
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: AlienVaultReputationFeed
+                accessor: Indicator.Rawjson
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: ef01415d-6ac6-4e16-8ab8-4e413645d416
+    type: title
+    task:
+      elasticcommonfields: {}
+      id: ef01415d-6ac6-4e16-8ab8-4e413645d416
+      version: -1
+      name: Done
+      description: Done
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 6e57e6a8-83a8-48f1-8136-19044c339538
+    type: regular
+    task:
+      elasticcommonfields: {}
+      id: 6e57e6a8-83a8-48f1-8136-19044c339538
+      version: -1
+      name: Delete Context
+      description: Delete field from context.
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    scriptarguments:
+      all:
+        simple: "yes"
+      index: {}
+      key: {}
+      keysToKeep: {}
+      subplaybook: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 735,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+sourceplaybookid: AlienVaultReputationFeed_Test
```

Comment:
Add `fromversion: 5.5.0` please.

Reply:
Check that it's written like this, please, because I wrote the comment from memory.